docs/lightning/04-salesforce-lambdas/04-contact-channel-analytics.md

```diff
@@ -168,9 +168,9 @@ Please note that your secret may also be formatted stored as a "Secret key/value
 
 2. For the **Name** and **Label**, enter AwsGenerateAudioRecordingURL.
 
-3. In the **URL** section, enter `https://lambda.{awsRegion}.amazonaws.com/2015-03-31/functions/{lambdaFunctionName}/invocations/`-- replace {awsRegion} with the awsRegion your serverless application resides in (for example, us-east-1), and replace {lambdaFunctionName} with the full name of the sfGenerateAudioRecordingStreaming lambda you recorded in the previous section.
```

Comment:
We need to remove the trailing slash (/) after "invocations" in the URL under SFDC Named Credentials. In order to import the call recording streaming on SFDC side.